WebContent/WEB-INF/jsp/watchList.jsp

```diff
@@ -703,7 +704,7 @@
                   <td style="width:1px; white-space:nowrap;">
                     <input type="checkbox" name="chartCB" id="p_TEMPLATE_ChartCB" value="_TEMPLATE_" checked="checked"
                             title="<fmt:message key="watchlist.consolidatedChart"/>"/>
-                    <tag:img png="icon_comp" title="watchlist.pointDetails"
+                    <tag:img png="icon_comp" id="p_TEMPLATE_pointDetails" title="watchlist.pointDetails"
```

Comment:
I would prefer there were no changes in the old UI because I cannot predict the consequences of such a change and it will be very difficult to improve the operation with the use of the dojo framework, different versions of jquery, and jquery plug-ins.

Reply:
Same situation like above.